config/config.tyndp.yaml

```diff
@@ -177,14 +177,17 @@ pypsa_eur:
   Store: []
 
 sector:
-  transport: false
+  transport: true
```

Comment:
I assume you included the transport sector to make the problem feasible. This creates an oil demand which is needed because of the forced supply of unsustainable bioliquids. 

An alternative would be to set the share of unsustainable bioliquids to zero instead, which should also make the problem feasible without requiring oil demand and all the additional network components.

However, the benchmark already seems to improve with the default PyPSA-Eur assumptions for land transport, shipping, and aviation. So I'm fine with keeping your current approach and adjusting the parameters to match TYNDP assumptions in a later PR.